test/e2e/framework/network_pods.go

```diff
@@ -16,14 +16,15 @@ const (
 // create a test pod inside the current test namespace on the specified cluster.
 // The pod will listen on TestPort over TCP, send sendString over the connection,
 // and write the network response in the pod  termination log, then exit with 0 status
-func (f *Framework) CreateTCPCheckListenerPod(cluster int, sendString string) *v1.Pod {
+func (f *Framework) CreateTCPCheckListenerPod(cluster int, sendString string, namespace string) *v1.Pod {
```

Comment:
please renane thus function to CreateTCPCheckListenerPodInNS and then create:
`
func (f *Framework) CreateTCPCheckListenerPod(cluster int, sendString string) *v1.Pod {
    CreateTCPCheckListenerPodInNs(cluster, sendString, f.Namespace)
}

so we avoid modifying existing tests or cluttering tests which don't need to deal with multiple namespaces.
`